src/cli/commands/version.js

```diff
@@ -130,6 +131,14 @@ export async function setVersion(
     return () => Promise.resolve();
   }
 
+  function buildCommitArgs(args): Array<string> {
+    args = args || ['commit'];
```

Comment:
Don't think allowing an empty args value is useful. It adds a new code path to be tested and currently this code path is not even being used. I'd rather us to enforce an array argument through proper flow definitions.